packages/sdk/src/mint/mint.ts

```diff
@@ -26,14 +43,28 @@ export type Splits = Record<string, number>;
 export const mint = (
   args: MintArgs,
 ): NearContractCall => {
-  const { nftContractId = DEFAULT_CONTRACT_ADDRESS, reference, ownerId, options = {}  } = args;
+  const {
+    nftContractId = DEFAULT_CONTRACT_ADDRESS,
+    metadata,
+    ownerId,
+    options = {},
+    noMedia = false,
+    noReference = false,
+  } = args;
 
   const { splits, amount, royaltyPercentage } = options;
-  
-  if (nftContractId == null) {
+
+  if (nftContractId === null) {
     throw new Error('You must provide a nftContractId or define a NFT_CONTRACT_ID environment variable to default to');
   }
 
+  if (!noReference && !metadata.reference) {
+    throw new Error('You must provide reference in your metadata or explicitly opt out of using reference');
```

Comment:
Looks like the conflict here moved these messages into constants which is nice, be good to follow that pattern as well

Reply:
Done